### DISCUSSION TITLE
Jasonelle v3 features requests

### DISCUSSION THREAD

**Discussion by DuckProjects** [jasonelle-archive/jasonelle-v2#121] [Ideas]
Hi all !

As previously announced by @clsource here : [Anouncement v2 EOL / new v3](https://github.com/jasonelle/jasonelle/discussions/120), Jasonelle will move forward and a v3 is in progress.

This v3 will be focused on iOS only (Android not planned, maybe in the future but there will be no promises).

**Do you think something was missing in Jasonelle v2 that would be a must-have in the v3 ? Please make your voice heard and submit your ideas and new features for v3 by replying in this thread.**

I will start with a new feature request : 
I would like v3 to generate a unique ID at app setup. It could be something like md5(timestamp.device_model.device_os).
This would be useful to get stats like unique visitors per app "page" when a user is not logged in.

Thoughts on this features ? Have a request ? Let's discuss it !

**Comment by clsource** [comment]
This is a list of features that the new version may have by default:

- URL Schemes: https://github.com/jasonelle/jasonelle/wiki/%5BHowto%5D-Deep-Linking-Jasonette
- Easier Push Notifications
- Localizable Strings
- Native HTTP
- Native Logger
- Color Helper
- node_modules compatible
- no cocoapods nor carthage
- iOS 14+
- Swift + Objc

**Comment by 5455230** [comment]
这是一个伟大的项目,感谢贡献者的付出和努力.
